app/core/src/main/java/com/thechance/core/data/datasource/ProductDataSourceImp.kt

```diff
@@ -28,12 +28,13 @@ import org.koin.core.component.KoinComponent
 class ProductDataSourceImp : ProductDataSource, KoinComponent {
 
     override suspend fun createProduct(
-        productName: String, productPrice: Double, productQuantity: String, categoriesId: List<Long>
+        productName: String, productPrice: Double, productQuantity: String, categoriesId: List<Long>, ownertId: Long
```

Comment:
Why the variable is named "ownerId", it's "marketId". And it's totally a different thing.